types/ldapjs/index.d.ts

```diff
@@ -75,8 +75,19 @@ export var Change: {
 	new(change: Change): Change;
 }
 
+export type SearchReference = any;
+
+export class SearchCallbackResponse extends EventEmitter {
```

Comment:
I would either make this an interface or not export it; otherwise, someone might import `SearchCallbackResponse` from the library and expect it to exist at runtime.